portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/clay/ClayAlertToast.testcase

```diff
@@ -53,7 +55,7 @@ definition {
 		}
 
 		task ("Then the alert does not disappear when hovered over") {
-            Pause(locator1 = "25000");
+            Pause(locator1 = "20100");
```

Comment:
I updated to just above 20 seconds to check if this will still pass. Not sure, if you tested this or if 25000 is minimum to get the test to pass on ci.

Reply:
The test passed, 🤔 but when I ran locally it works with the 20 seconds after the mouse is removed, I think we can put a little more time and test, maybe `Pause(locator1 = "21000");`? 

https://user-images.githubusercontent.com/44584050/209683920-63694ca5-7adc-4037-a8fb-ef0db6e1fa23.mp4

Reply:
Two points to consider:

1. The test is to check that the message does not autoclose when mouse hovers over the alert.

2. In your screencapture, your mouse hovers over the alert for ~5seconds which extends the autoclose timer by 5 seconds. Hence, why the alert autocloses after 25 seconds in your recording.

Reply:
I tested a negative test to validate at 19 seconds and 21.1 seconds. It seems to pass consistently when I tested locally. I included this as a baseline to help sanity check the original scenario is valid.